tests/coverage/async_closure.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(coverage_attribute)]
```

Comment:
Nit: `#![feature(coverage_attribute)]` isn't needed here, as there are no `#[coverage(..)]` attributes in the source.